c2cciutils/checks.py

```diff
@@ -247,11 +285,16 @@ def workflows(
     args: Namespace,
 ) -> bool:
     """
-    Do some generic check on the workflows
+    Do some generic check on the workflows.
```

Comment:
```suggestion
    Check each workflow have a timeout and/or do not use blacklisted images.
```
Sorry but I do not know what is a generic check on a workflow.